GLTFSDK/Inc/GLTFSDK/MeshPrimitiveUtils.h

```diff
@@ -5,15 +5,15 @@
 
 #include <vector>
 
+#include <GLTFSDK/GLTF.h>
+
 namespace Microsoft
 {
     namespace glTF
     {
         class Document;
         class GLTFResourceReader;
-        struct Accessor;
-        struct MeshPrimitive;
-        struct MorphTarget;
+        class BufferBuilder;
```

Comment:
Can the BufferBuilder forward declaration be removed now?

Reply:
Good point. Cleaning it up now.